azavea/raster-foundry
# Feature/cmb/add fonts use yarn

## Overview

This commit switches use of NPM to yarn to hopefully solve some of our node package dependency woes. Additionally, I was able to update the fonts (_I_ _think_) so styles are ready to be updated.

@designmatty - we can add a commit with your font changes to this PR if you would like.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~

## Testing Instructions

 * Run `./scripts/setup`
 * Install yarn after switching to node 6 with nvm (`nvm use 6`): `npm install -g yarn`
 * Start server and frontend, verify no errors are thrown by webpack
